Podfile

```diff
@@ -18,35 +20,35 @@ pod 'Mixpanel', '2.8.2'
 pod 'CocoaLumberjack', '= 2.0.0'
 pod 'NSLogger-CocoaLumberjack-connector', :git => 'https://github.com/steipete/NSLogger-CocoaLumberjack-connector.git', :tag => '1.5'
 pod 'google-plus-ios-sdk', '~>1.5'
-pod 'CrashlyticsLumberjack', '2.0.0'
```

Comment:
Yep. I added a comment to https://github.com/wordpress-mobile/WordPress-iOS/commit/26cdc21211348a719425f510547adf0b54be7f3b but pasting it here as well for reference

From CocoaPods/CocoaPods#3194 (comment)

> ...if a library is not dynamic and you're trying to use it with
> frameworks you can use it, but you cannot use it as a dependency of
> something else via CocoaPods.

We can't have CrashlyticsLumberjack because it depends on Crashlytics.
Since it's only a few lines of code, it's just easier to integrate that
code in the app than to use ugly workarounds.